cpp/onedal/backend/interop/common.hpp

```diff
@@ -52,7 +53,10 @@ inline auto call_daal_kernel(const context_cpu& ctx, Args&&... args) {
     return dal::backend::dispatch_by_cpu(ctx, [&](auto cpu) {
         constexpr daal::CpuType daal_cpu_type = daal::CpuType::avx;
         // constexpr daal::CpuType daal_cpu_type = get_daal_cpu_type(cpu);
-        return CpuKernel<Float, daal_cpu_type>().compute(args...);
+        auto status = CpuKernel<Float, daal_cpu_type>().compute(args...);
+        if (!status) {
+            dal::backend::convert_status_to_exception(status);
+        }
```

Comment:
It is a good thought to localize error conversion in one place - but I'm afraid this does not work. Do not we need to make a specific conversion for different algorithms and situations? For example, when PCA fails with some error code, it needs to be converted to the first exception, but when LogReg fails with the same code - another exception shall be raised.
For me, it is not obvious that interop call of `call_daal_kernel()` will throw an exception. From my perspective, it shall return 'Status', which can be converted after calculation (inside [backend implementation](https://github.com/mchernov-intel/daal/blob/error-handling/cpp/onedal/decomposition/pca/backend/cpu/train_kernel_cov.cpp), for example)